app/scripts/services/websocket.js

```diff
@@ -80,8 +80,30 @@ services.service('KodiWS', ['$q', '$window', 'KODI_URL', 'KODI_PORT', 'SOCKET_TI
     return deferred.promise;
   }
 
+  /**
+   *
+   * @param thumbUrl {string} - encoded url for thumbnail (as it usually return Kodi), for example: <code>image://http%3a%2f%2fcdn-radiotime-logos.tunein.com%2fs100934q.png/</code>
+   * @return url {string} - proper thumbnail url, for example: <code>http://192.168.2.124:8080/image/http%3a%2f%2fcdn-radiotime-logos.tunein.com%2fs100934q.png</code>
+   */
+  function getThumbnailURL(thumbUrl) {
+    if (!thumbUrl)
+      return thumbUrl;
+    thumbUrl = thumbUrl.replace('image://', '');
+    if (thumbUrl.charAt(thumbUrl.length - 1) === '/') {
+      thumbUrl = thumbUrl.substring(0, thumbUrl.length - 1);
+    }
+    if (thumbUrl.charAt(0) === '/') {
```

Comment:
That doesn't seem to remove it actually. I think it's because it is URL encoded at this point, so it's never a `/` here. Maybe do this after decodeURIComponent?